src/data/machines.js

```diff
@@ -141,173 +141,172 @@ export const ephemeralExternalIpAddressPrice = { // per hour in dollars for all
 // Disk prices below are per GB per https://cloud.google.com/compute/all-pricing#disk
 // GPU prices below are hourly and per GPU (https://cloud.google.com/compute/gpus-pricing).
 // The data below comes from Google's pricing API: https://cloud.google.com/billing/v1/how-tos/caltalog-api
-// There is a script available in https://gist.github.com/wnojopra/be5be3ab7e6a09a351b89085992a940d
-// that parses the API and outputs this data.
-// TODO: Update the above URL in the comment to the proper script location
+// There is a script available at https://github.com/broadinstitute/dsp-scripts/blob/master/terra-ui/regionality/parse_pricing_api.py.
+// It parses the API and outputs the response data.
 export const regionToPrices = [
   {
-    name: 'ASIA-EAST1', monthlyDiskPrice: 0.04,
+    name: 'ASIA-EAST1', monthlyDiskPrice: 0.04, monthlySSDPrice: 0.17,
```

Comment:
It may be clearer and more consistent to use the keys `monthlyStandardDiskPrice` and `monthlySSDDiskPrice` (`D` in `SSD` stands for `Drive` and not `Disk`).

Reply:
Yes that's true, it will require going in and changing the script again and I can check in with Willy about the change if you think it would be best! When changing the script I wanted to keep the changes more minimal and just add the SSD cost value without renaming other values in the output because I did not initially know how the output was being used.

Reply:
Yes, I think it'd be good to prioritize readability here.

> I wanted to keep the changes more minimal and just add the SSD cost value without renaming other values in the output because I did not initially know how the output was being used

Good idea to check use cases first. Searching for `monthlyDiskPrice`, it's only referred to once in `runtime-utils.js` so I can't see any issues with renaming it.